apps/web/app/app.dub.co/(onboarding)/onboarding/(steps)/products/product-selector.tsx

```
@@ -0,0 +1,113 @@
+    description: "Short link management, analytics, QR codes, and tracking.",
+    paidPlanRequired: false,
+  },
+  partners: {
+    image: "https://assets.dub.co/icons/trophy.webp",
+    title: "Dub Partners",
+    description: (
+      <>
+        Manage a partner program with tracking, plus{" "}
+        <a
+          href="https://dub.co/links"
+          target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)